modules/ROOT/pages/to-find-info.adoc

```diff
@@ -17,23 +17,57 @@ text with an All Types filter.
 * RAML Fragments - A RAML document that has a version and an identifier, but is not in itself a complete RAML specification.
 * Custom - Blogs, articles, and more - Links that assist with knowledge sharing.
 
-== About Asset Details
+== Asset Details
 
-When you click an asset, Exchange provides detail information:
+When you click an asset, Exchange provides detailed information:
 
 * Left navigation area - Lists additional pages, saved search terms, application names, instances, and for a REST API,
 the HTTP functions in the REST specification.
 * Right detail panel - Lists the asset type, when the asset was created and by which organization, when the asset was last published, the version, and the asset's dependencies. For REST APIs, the right panel lets you test the API by creating example calls, test data (mocking), and the ability to send data to the API and view the results.
 
-== About Exchange Search
+== Exchange Search
 
 Search lets you find Exchange assets that contain one or more terms
 in an asset title, asset ID, or a tag in an asset. You can also combine search terms with the filters listed in All Types.
 Searches only apply to the Exchange you are viewing.
 If you search in a private Exchange, the search only lists the
 private Exchange assets and not public Exchange assets.
 
-== To Search for Phrases
+== Save a Search
+
+. Enter a search term and click Save This Search.
+. Exchange prompts you to give the search a name, and to select if the search is for your organization or personal. Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
+. View the left navigation bar to see the saved searches. Exchange lists search terms specific to your business group. Below those terms are saved searches you created.
++
+image::ex2-saved-searches-groups.png[Saved Search Groups]
+
+== View Assets by Business Group
+
+A business group provides a way to group individuals within an organization. A system administrator assigns users to belong to a business group. Each business group can have separate Exchange assets. You need to log into
+Anypoint Platform to view the business groups. Click a group name from the taskbar.
+
+image::ex2-biz-groups.png[Screenshot - Business groups in upper taskbar drop-down menu]
+
+If you cannot see additional groups, contact your Exchange administrator. 
+
+If you cannot find an asset in one group, it may be in another business group, or click *Assets* to search other assets. The left navigation area provides additional areas to search. The MuleSoft scope lists assets provided by MuleSoft.
```

Comment:
If you cannot find an asset in one group, it may be in another business group. You can find assets from other business groups by clicking on *Assets* to search for assets across all organizations. The left navigation area provides additional areas to search. 
- The MuleSoft scope lists assets provided by MuleSoft.
- The "Assets" scope lists assets provided by Mulesoft and your own master organization.
- The <Business Group> scope lists assets built by users within that Business group"

Goal is to 
1) Tell them how & where they can find/search for assets across all Business Groups
2) Define the various org scopes in the left panel

Feel free to change the wording, but make sure that those goals are achieved.